pyproject.toml

```diff
@@ -1,7 +1,44 @@
 [build-system]
-requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
+requires = ["setuptools>=61", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pygmt"
+description = "A Python interface for the Generic Mapping Tools"
+readme = "README.rst"
+requires-python = ">=3.8"
+license = {text = "BSD License"}
+authors = [{name = "The PyGMT Developers", email = "leouieda@gmail.com"}]
+maintainers = [{name = "The PyGMT Developers", email = "leouieda@gmail.com"}]
```

Comment:
Probably should stop using @leouieda's email here. Don't we have a proper PyGMT dev email now @meghanrjones?

Reply:
Not that I am aware of.

Two options:
1. Setup a listserve with the maintainers' email addresses. We could probably do this through UH but there are some sustainability issues there.
2. Register a new gmail (or other email provided) account and share login information.

I might actually prefer option 1 if we can find a stable alternative to UH listserv, because I would rather have emails forwarded to an existing address than have another email address to check.

Reply:
> Probably should stop using @leouieda's email here.

+1 to that 🙂 

@meghanrjones I recently made one for Fatiando for this. You can setup auto forwarding in Gmail to other addresses so you wouldn't have to check it.

Reply:
Thanks for the info. I created pygmt.team@gmail.com.

Reply:
I've updated the email address in cc9d8e0d23de4f888a7c17466572d21639b4fd02.